coreax/solvers/recombination.py

```diff
@@ -320,10 +340,14 @@ def _co_linearize(
     non_zero_weights_mask = weights > 0
```

Comment:
This functions jaxtyping needs spaces before the `"n m"`

Reply:
Is this a style thing, or a complaint from a linter? In some places I've prefixed with a space, in others I haven't.

Reply:
Started as a way to avoid ruff complaining but then maintained for style, but actually probably better to avoid the space where it is not required